deploy/osd-suricata/20-osd-suricata-Daemonset.yaml

```diff
@@ -33,7 +33,7 @@ spec:
         - mountPath: /host/
```

Comment:
Change to `/host/var/log` and fix the volume to point to `/var/log`

Reply:
Updated